Update hu.json

FUst1ke opened the pull request:
Added new tranlations for the new discord-based whitelist, and made some grammar fixes :)

FUst1ke committed:
Update hu.json

Added new tranlations for the new discord-based whitelist, and made some grammar fixes :)

wowjeeez commented on locale/hu.json:
```diff
@@ -17,40 +17,40 @@
+            "label_reason": "Kitiltás indoka",
             "label_id": "Ban ID",
-            "note_multiple_bans": "Note: you have more than one active ban on your identifiers."
+            "note_multiple_bans": "Figyelem: Több mint egy kitiltásod van ezen a fiókon."
```
I think this can be quite misleading, the translation should include that player has more than 1 *active* ban.
Proposal:
`Figyelem: Több mint egy aktív kitiltásod van ezen a fiókon.`

wowjeeez commented on locale/hu.json:
```diff
@@ -17,40 +17,40 @@
-            "deny_title": "You are required to join our Discord Guild to connect.",
-            "deny_message": "Please join the guild %{guildname} then try again."
+            "mode_title": "Ez a szerver <strong>Discord szerver tag whitelist</strong> módban van.",
+            "insufficient_ids": "A <code>discord</code> azonosítód nem elérhető, ami szükséged az ellenőrzéshez, hogy fent vagy-e a szerveren. Kérlek nyisd meg a discordot és próbáld meg újra.",
```
`szükséges`, `Discordot`, `Discord`, `annak az ellenőrzéséhez`

wowjeeez commented on locale/hu.json:
```diff
@@ -17,40 +17,40 @@
-            "deny_noroles_title": "You do not have a whitelisted role required to join.",
-            "deny_noroles_message": "To join this server you are required to have at least one of the whitelisted roles on the guild %{guildname}."
+            "mode_title": "Ez a szerver <strong>Discord whitelist rang</strong> módban van.",
+            "insufficient_ids": "A <code>discord</code> azonosítód nem elérhető, ami szükséged az ellenőrzéshez, hogy fent vagy-e a szerveren. Kérlek nyisd meg a discordot és próbáld meg újra.",
```
exact same wording as above

wowjeeez commented on locale/hu.json:
```diff
@@ -17,40 +17,40 @@
+            "mode_title": "Ez a szerver <strong>License Whitelist</strong> módban van.",
+            "insufficient_ids": "Nincs <code>license</code> azonosítód, ami azt jelenti, hogy a szerveren az <code>sv_lan</code> be van kapcsolva. Ha te vagy a tulajdonos, a <code>server.cfg</code> fájlban ki tudod kapcsolni.",
+            "deny_title": "Nem vagy engedélyezve a szerveren.",
             "request_id_label": "Request ID"
```
`Kérés ID`, geci furán hangzik de lekell fordítani sajna

wowjeeez commented on locale/hu.json:
```diff
@@ -107,8 +107,8 @@
-                    "label": "Toggle super jump mode, the player will also run faster",
-                    "success": "Super Jump enabled"
+                    "label": "Magas ugrás és gyors futás",
+                    "success": "Super Jump engedélyezve"
```
`bekapcsolva` maybe?

FUst1ke replied:
azért írtam engedélyezve-re, mert ha megnézed a fölötte lévőket azok is úgy vannak, mondom akkor legyen rendszer nézzen ki ugyan úgy.

wowjeeez replied:
jogos

wowjeeez commented on locale/hu.json:
```diff
@@ -166,10 +166,10 @@
+                    "label": "Boostold a járművedet!",
+                    "success": "Jármű boostolva!",
+                    "already_boosted": "Ezt a járművet már boostoltad.",
+                    "unsupported_class": "Ez a jármű típus nem támogatott."
```
`Jármű típus` is one word, `Járműtípus`

wowjeeez reviewed (changes requested):
Looks good overall, please fix the annotated changes and thank your for your contribution!

FUst1ke committed:
Update hu.json -2nd version

Did the changes which was menitoned in the last pull rq. Hope this will be fine :)

wowjeeez reviewed (dismissed):
woo

FUst1ke committed:
godmode fix

Having issues in esx legacy 1.9 with this solution, but with this it works fine and doesnt have any ms- change (still 0.00 is you use this)

tabarra commented:
Thanks for the PR!
Since this PR also included a "godmode fix" i had to cherry pick the two translation commits and will close this pr.
